pkg/load/configAgent.go

```diff
@@ -94,10 +102,12 @@ func (a *agent) loadFilenameToConfig() error {
 		if info != nil && !info.IsDir() && (ext == ".yml" || ext == ".yaml") {
 			configSpec, err := Config(path)
 			if err != nil {
+				a.recordError("failed to load config")
 				return fmt.Errorf("failed to load ci-operator config (%v)", err)
 			}
 
 			if err := configSpec.ValidateAtRuntime(); err != nil {
+				a.recordError("invalid config")
```

Comment:
"invalid ci-operator config" would carry some context and is reasonably future-proof